src/main/java/InputBoundary/playAudioInputBoundary.java

```diff
@@ -1,15 +1,12 @@
 package InputBoundary;
 
-import InputData.audioInputData;
-import javazoom.jl.decoder.JavaLayerException;
-
-import java.io.FileNotFoundException;
+import InputData.songInputData;
 
 /**
  * Interface for the playAudioInteractor to implement for the use case
  */
 public interface playAudioInputBoundary {
 
-    void playAudio(audioInputData songFile);
+    void playAudio(songInputData song);
```

Comment:
Just to clarify, is this the method that I will be calling? (playAudio vs. playSong)